.github/workflows/connector-performance-command.yml

```diff
@@ -180,6 +181,7 @@ jobs:
           export STREAM_NUMBER=$STREAM_NUMBER
           export SYNC_MODE=$SYNC_MODE
           export HARNESS=$HARNESS_TYPE
+          export DD_API_KEY=$DD_API_KEY
```

Comment:
Maybe I'm missing something here, but why do we need to export like this when the env var is already set?

Reply:
I’m following the comment on line 178 - seems we need to export those to be accessible to envsubst- though I haven’t tested if that comment is true in this case

Reply:
you are right - that comment wasn't true. Fixed and cleaned it up.

Reply:
and it still works! https://github.com/airbytehq/airbyte/actions/runs/6540161486/job/17759616382 so you are right, we don't need to re-export stuff that has already been set in env.